TESTING_WITH_IAF-TEST.md

```diff
@@ -31,8 +31,8 @@ The module's test scenarios can be run manually with the Larva testtool. This wi
    - `credentialFactory.class=nl.nn.credentialprovider.PropertyFileCredentialFactory`
    - `authAliases.expansion.allowed=testalias`
 2. In the Tomcat Overview window, set the port number for HTTP/1.1 to 80. If you wish to use another port, please set the property `web.port` accordingly in catalina.properties.
-3. In the same window, go to the Modules tab at the bottom and add "/iaf-test" as a web module.
-4. In the Project Explorer, right-click the iaf-test module and select Properties. Go to Deployment Assembly, press [ Add... ]. Select Folder, press [ Next ]. Select the src/main/configurations folder, and press Finish. In the text field right of your new src/main/configurations item, enter `WEB-INF/classes/configurations`. _(might not be necessary anymore)_
+3. In the same window, go to the Modules tab at the bottom and add "/iaf-test" as a web module (has tool-tip ibis-adapterframework-test).
+4. In the Project Explorer, right-click the ibis-adapterframework-test project and select Properties. Go to Deployment Assembly, press [ Add... ]. Select Folder, press [ Next ]. Select the src/main/configurations folder, and press Finish. In the text field right of your new src/main/configurations item, enter `WEB-INF/classes/configurations`. _(might not be necessary anymore)_
```

Comment:
Afhankelijk hoe je de modules importeert in Eclipse krijgt het de juiste (opgegeven iaf-test) naam, of de verkeerde (afgeleid uit de pom ibis-adapterframework-test) naam. Ik vind het een beetje onzinnig dit aan te passen.

Zie de project settings bestanden (`.project`) voor de juiste naam.

Reply:
Verwerkt.